kinto/tests/test_views_disable_default.py

```diff
@@ -0,0 +1,47 @@
+import webtest
+from pyramid.config import Configurator
+
+from cliquet.tests import support as cliquet_support
+from kinto import main
+
+from .support import (BaseWebTest, unittest, get_user_headers,
+                      MINIMALIST_BUCKET, MINIMALIST_COLLECTION,
+                      MINIMALIST_RECORD)
+
+
+class DisableDefaultBucketViewTest(BaseWebTest, unittest.TestCase):
+
+    test_url = '/buckets/default'
+
+    def setUp(self):
+        super(DisableDefaultBucketViewTest, self).setUp()
+        self.events = []
```

Comment:
you don't need to override `setUp`, `tearDown`, `_get_test_app` or `get_app_settings` since you don't change their behaviour